docs/contracts.rst

```diff
@@ -982,8 +982,8 @@ contract, and a regular ``JUMP`` call will be used instead of a ``DELEGATECALL``
 .. index:: using for, set
 
 The following example illustrates how to use libraries (but
-be sure to check out :ref:`using for <using-for>` for a
-more advanced example to implement a set).
+be sure to check out `using-for <https://solidity.readthedocs.io/en/latest/contracts.html#using-for>`_ 
```

Comment:
I don't think `solidity.readthedocs.io` should appear in the documentation.  An older version of this manual should not link to the newest version like this.

Before this change, [the link is working already](https://solidity.readthedocs.io/en/latest/contracts.html#libraries).